[2023-12-10T07:07:45.601+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: taskflow.load_Orders_to_bigquery manual__2023-12-10T07:04:37.336852+00:00 [queued]>
[2023-12-10T07:07:45.676+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: taskflow.load_Orders_to_bigquery manual__2023-12-10T07:04:37.336852+00:00 [queued]>
[2023-12-10T07:07:45.679+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-12-10T07:07:45.783+0000] {taskinstance.py:1382} INFO - Executing <Task(GCSToBigQueryOperator): load_Orders_to_bigquery> on 2023-12-10 07:04:37.336852+00:00
[2023-12-10T07:07:45.833+0000] {standard_task_runner.py:57} INFO - Started process 1610 to run task
[2023-12-10T07:07:45.883+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'taskflow', 'load_Orders_to_bigquery', 'manual__2023-12-10T07:04:37.336852+00:00', '--job-id', '154', '--raw', '--subdir', 'DAGS_FOLDER/test_decorator.py', '--cfg-path', '/tmp/tmpjhhlzpwb']
[2023-12-10T07:07:45.935+0000] {standard_task_runner.py:85} INFO - Job 154: Subtask load_Orders_to_bigquery
[2023-12-10T07:07:46.545+0000] {task_command.py:416} INFO - Running <TaskInstance: taskflow.load_Orders_to_bigquery manual__2023-12-10T07:04:37.336852+00:00 [running]> on host 580ab8862310
[2023-12-10T07:07:49.040+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='taskflow' AIRFLOW_CTX_TASK_ID='load_Orders_to_bigquery' AIRFLOW_CTX_EXECUTION_DATE='2023-12-10T07:04:37.336852+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-10T07:04:37.336852+00:00'
[2023-12-10T07:07:49.457+0000] {connection.py:232} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2023-12-10T07:07:49.512+0000] {base.py:73} INFO - Using connection ID 'google_cloud_datastore_default' for task execution.
[2023-12-10T07:07:49.603+0000] {gcs_to_bigquery.py:376} INFO - Using existing BigQuery table for storing data...
[2023-12-10T07:07:49.617+0000] {credentials_provider.py:353} INFO - Getting connection using `google.auth.default()` since no explicit credentials are provided.
[2023-12-10T07:07:50.384+0000] {bigquery.py:2311} INFO - Project is not included in destination_project_dataset_table: dataset_for_dag_testing.Orders; using project "northwind-qu"
[2023-12-10T07:07:50.401+0000] {gcs_to_bigquery.py:380} INFO - Executing: {'load': {'autodetect': True, 'createDisposition': 'CREATE_IF_NEEDED', 'destinationTable': {'projectId': 'northwind-qu', 'datasetId': 'dataset_for_dag_testing', 'tableId': 'Orders'}, 'sourceFormat': 'CSV', 'sourceUris': ['gs://us-central1-test-env-b4f32c38-bucket/data/Orders.csv'], 'writeDisposition': 'WRITE_TRUNCATE', 'ignoreUnknownValues': False, 'skipLeadingRows': 1, 'fieldDelimiter': ',', 'quote': None, 'allowQuotedNewlines': False, 'encoding': 'UTF-8'}}
[2023-12-10T07:07:50.499+0000] {bigquery.py:1595} INFO - Inserting job ***_taskflow_load_Orders_to_bigquery_2023_12_10T07_04_37_336852_00_00_f4f2d2cd86384bc4a4365ede85e523bf
[2023-12-10T07:07:56.085+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=taskflow, task_id=load_Orders_to_bigquery, execution_date=20231210T070437, start_date=20231210T070745, end_date=20231210T070756
[2023-12-10T07:07:56.171+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-10T07:07:56.245+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
